### PR TITLE
Feat : Success story section for backend Page

### PR DESCRIPTION
**Pull request by cp-ashish-k**
## Review checklist

**Note:** Make sure all points should be checked before submiting PR for review, otherwise PR will be cosidered invalid.

- [x] Follow guidelines for creating branch name, adding commit messages or PR title
- [x] Add MR description if applicable
- [ ] Follow guidelines for naming conventions everywhere (i.e files/folders, data structures)
- [x] Reuse code (if the same code is written twice, make it common and reuse it at both places)
- [x] Remove unused or commented code if not required
- [x] Make MR self-approved - review MR as a reviewer and give it self-approval if everything is ok, if not make the required changes
- [ ] In progress MR should be marked as draft
- [ ] Make MR mark as ready before submitting it for review
- [x] Add images, videos, or gifs of UI
- [x] Add page speed score screenshot of UI (It should be > 85 for mobile and > 90 for desktops)

**Design should be tested on:**

- [x] Inspector (all iPhones and android devices like samsung, motorola)
- [x] Desktop (chrome, firefox, safari)
- [ ] Laptop / Macbook (chrome, firefox, safari)
- [x] iPhone(safari)
- [x] Available android devices (chrome, firefox)
- [x] iPad(safari)
- [x] Tablet (chrome, firefox)

 - Pagespeed Insight score for mobile
image

- Pagespeed Insight score for desktop
image

- ScreenShot for mobile UI
image

- ScreenShot for desktop UI
image

### CODE REVIEW

**Review by cp-jagruti-a — APPROVED** [review]

File: nuxt-frontend/tailwind.config.js [hunk 1]
```diff
@@ -36,9 +36,11 @@ module.exports = {
           900: "#3d3d3d",
           87: "#000000DE",
           60: "#00000099",
+          bg: "#121212",
```
Comment (cp-dharti-r) [inline nuxt-frontend/tailwind.config.js:39]: let's not use it in this way, if it's a shade of the black color then give some meaningful name.

File: nuxt-frontend/components/backend-development/SuccessStorySection.vue [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+    designation: "CTO at Udini, France",
+    review: `"Multiple versions of this Android app have been successfully delivered over time. They are always very responsive on bug resolution. They are very efficient at producing complex interfaces and  high quality apps."`,
+  },
+];
+</script>
+<style lang="postcss" scoped>
+.swiper-slide-active {
+  @apply !scale-100;
+}
+.client {
```
Comment (cp-dharti-r) [inline nuxt-frontend/components/backend-development/SuccessStorySection.vue:174]: it should be `slide` only if it hasn't a default class with the same name

File: nuxt-frontend/tailwind.config.js [hunk 3]
```diff
@@ -36,9 +36,11 @@ module.exports = {
           900: "#3d3d3d",
           87: "#000000DE",
           60: "#00000099",
+          darktone: "#121212",
```
Comment (cp-dharti-r) [inline nuxt-frontend/tailwind.config.js:39]: it's not a shade of black, so remove it from here and define new with a `deep-charcoal` name

File: nuxt-frontend/tailwind.config.js [hunk 4]
```diff
@@ -59,6 +60,9 @@ module.exports = {
           200: "#e2e2e2",
           400: "#8d8a8a",
         },
+        deep: {
```
Comment (cp-dharti-r) [inline nuxt-frontend/tailwind.config.js:63]: it should be simply `deep-charcoal: "#121212",`

**Review by cp-dharti-r — APPROVED** [review]

**Review by cp-jagruti-a — APPROVED** [review]

**Pull request merged**